build.gradle.kts

```diff
@@ -208,7 +245,16 @@ fun Test.sharedIntegrationTestConfig(buildCodyDir: File, mode: String) {
   include("**/AllSuites.class")
 
   val resourcesDir = project.file("src/integrationTest/resources")
+
+  jvmArgs(
+    "--add-opens", "java.desktop/java.awt=ALL-UNNAMED",
+    "--add-opens", "java.desktop/javax.swing=ALL-UNNAMED",
+    "--add-opens", "java.desktop/sun.awt=ALL-UNNAMED",
+    "--add-opens", "java.base/java.lang=ALL-UNNAMED"
+  )
+
   systemProperties(
+      "java.system.class.loader" to "com.intellij.util.lang.PathClassLoader",
```

Comment:
Why do we need this?

Reply:
I have added this setting because otherwise the integration tests were failing. But now it looks like it isn't actually necessary.